plugin-core/src/main/java/appland/cli/DefaultCommandLineService.java

```diff
@@ -476,18 +476,23 @@ private static boolean isSupported() {
                 || SystemInfo.isWindows && CpuArch.isIntel64();
     }
 
-    private static @NotNull KillableProcessHandler startProcess(@NotNull Path workingDir,
+    public static @NotNull KillableProcessHandler startProcess(@NotNull Path workingDir,
```

Comment:
nit: it would be enough to make this package-visible to support testing

Reply:
Thanks, my Java is a bit rusty!